daemon/web/styles/typography.sass

```diff
@@ -0,0 +1,50 @@
+/* Text styles */
+
+=Paragraph()
+  font-family: Rubik
+  font-size: 10px
+  color: #ffffff
+
+
+h3
+  font-family: Rubik
+  font-size: 14px
+  font-weight: 500
+  font-style: normal
+  font-stretch: normal
+  line-height: normal
+  letter-spacing: normal
+  color: $darkgrey
+
+
+h4
+  font-family: Rubik
+  font-size: 14px
+  font-weight: normal
+  font-style: normal
+  font-stretch: normal
+  line-height: normal
+  letter-spacing: normal
+  color: $greyblue
+
+
+h2
```

Comment:
maybe nitpicking but it might be nice if these were in order (`h1` > `h2` > `h3` > ...)

Reply:
on that topic, most of the traits on these styles are the same - I think you can fiddle around with inheritance a bit to bring down the verbosity. @PiggySpeed can probably correct me on this, but it might go:

```sass
%header-base
   font-family: Rubik
   font-style: normal
   font-stretch: normal
   line-height: normal

h3
   @extend header-base
   font-size: 14px
   font-weight: 500
   letter-spacing: normal
   color: $darkgrey

...
```

Reply:
I would also suggest mixins as there are quite a few shared properties.